src/main/kotlin/org/jmailen/gradle/kotlinter/KotlinterPlugin.kt

```diff
@@ -2,13 +2,14 @@ package org.jmailen.gradle.kotlinter
 
 import com.android.build.gradle.BaseExtension
 import com.android.build.gradle.api.AndroidSourceSet
-import java.io.File
+import com.android.build.gradle.internal.tasks.factory.dependsOn
```

Comment:
This will cause non-android projects to fail. You'll need to use the generic dependency setting, see below